### ISSUE TITLE
Version mismatch on search and package

### ISSUE THREAD

**Issue by Rycochet** [npm/registry-issue-archive#174]
Backstory - MathJAX shut down their CDN, and released a bugfixed version (2.7.1) removing references to the old CDN. JSDelivr automatically scrapes NPM for the latest version - but was still showing the previous (2.7.0).

Search page - https://www.npmjs.com/search?q=mathjax
```2.7.0```

Package page - https://www.npmjs.com/package/mathjax
```2.7.1```

Possibly related to #100.

**Comment by MartinKolarik** [comment]
When you say jsDelivr is showing 2.7.0, do you mean this link https://www.jsdelivr.com/projects/mathjax ? jsDelivr website doesn't list npm packages yet, you can get a listing for npm packages at https://cdn.jsdelivr.net/npm/mathjax/ (which should show 2.7.1).

**Comment by Rycochet** [comment]
Yep - the github repo is also on 2.7.1 (https://github.com/mathjax/MathJax/releases since the 25th of April) - hence where (and I'm probably wrong) it made it look like jsDelivr wasn't updating automatically, and where I ended up noticing that the search page had a version mismatch with the package page (with jsDelivr matching the search page and not the package or repo it's an obvious correlation, even if wrong) ;-)

**Comment by MartinKolarik** [comment]
I see... As I said, jsDelivr website doesn't list packages from npm yet, but they are always available immediately (within one minute after publishing), so this is not related to the npm version mismatch.

**Comment by Rycochet** [comment]
@MartinKolarik Except apparently the MathJax package was published 3 months ago, and jsDelivr is still showing the previous version - separate issue to this though, as this is about npmjs's caching of version numbers ;-)

**Comment by bcoe** [comment]
@Rycochet looking at our search indexer, the issue seems to be that the tarball has enough files in it that it's flagging a "max files to untar" check we have when attempting to untar the file. I see that there are `3222` files in the package, it might just be that we need to loosen this restriction a bit.

**Comment by silkentrance** [comment]
@bcoe looking at https://www.npmjs.com/search?q=mathjax and https://www.npmjs.com/package/mathjax it shows that newer versions have been picked up since. However, the version shown on the search page and the version shown on the package page are still off.

**Comment by silkentrance** [comment]
@bcoe perhaps you should not extract every file from the tar but instead limit it to only 'package.json'? at least for figuring out the most recent version of the package.
